scripts/generate-proto.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
I see this follows the pattern of gribi, but the more common pattern right now is to name this file "compile_protos.sh" and move it up a directory.

Reply:
thanks, addressed